app/controllers/mentoring/automation_controller.rb

```diff
@@ -12,6 +12,7 @@ def with_feedback
   def edit
     @representation = Exercise::Representation.find_by!(uuid: params[:uuid])
     @examples = Exercise::Representation::FindExampleSubmissions.(@representation)
+    @automation_params = params.permit(:order, :criteria, :page, :track_slug, :only_mentored_solutions)
```

Comment:
Can we nest these down a level in some key pls. Maybe `source`?
```suggestion
    @automation_params = params.permit(source: {:order, :criteria, :page, :track_slug, :only_mentored_solutions})` 
```

(I’m not certain that syntax is correct)